pkg/api/customization/monitor/cluster_graph_action.go

```diff
@@ -69,7 +73,16 @@ func (h *ClusterGraphHandler) QuerySeriesAction(actionName string, action *types
 	var svcName, svcNamespace, svcPort, token string
 	if inputParser.Input.Filters["resourceType"] == "istiocluster" {
 		inputParser.Input.MetricParams["namespace"] = ".*"
-		svcName, svcNamespace, svcPort = monitorutil.IstioPrometheusEndpoint()
+		selector := fields.OneTermEqualSelector("metadata.name", monitorutil.IstioAppName)
+		app, err := h.appGetter.Apps("").List(metav1.ListOptions{FieldSelector: selector.String()})
```

Comment:
Why using `List` function?

Reply:
Since `App` is namespaced scope and we can not get the `System` projectId from the input, so `List` function is more applicable.

Reply:
There is a change that user use the same name as `istioAppName`. We don't know what problem it would cause if you use name to list it.
Also you are using `appInterface` with all namespace scope, it will list all the istio app in your setup. Even there is in different cluster.

Reply:
Updated.